src/pages/index.tsx

```diff
@@ -30,39 +31,71 @@ const Cards = () => (
       >
         {[
           {
-            header: "DevOps, monitoring and observability",
-            content:
-              "Collect CPU, memory and storage metrics from your infrastructure and get real-time visibility into your entire stack.",
+            header: "Financial market data",
+            content: (
+              <p>
+                Store years of market tick data to identify historical trends,
+                and correlate it with orders and executed trades via{" "}
+                <Link to="/docs/reference/sql/join/#asof-join">ASOF joins</Link>.
+                Monitor market activity in real-time, compute time-series charts
+                and price candles on the fly. Open source kdb+ alternative.
+              </p>
+            ),
           },
 
           {
-            header: "Financial market data",
-            content:
-              "Store market tick data to identify historical trends, find correlations and analyze trades in real-time. Build aggregated views across multiple venues and efficiently compute live order books.",
+            header: "Application metrics",
+            content: (
+              <p>
+                Track and visualize user behavior data, API calls, data latency,
+                and other application events and analytics in real-time.
+              </p>
+            ),
```

Comment:
No need to wrap with element, if `content` is just a `string`:
```suggestion
            content:
              "Track and visualize user behavior data, API calls, data latency, and other application events and analytics in real-time.",
```